### PR TITLE
CRM-17004 - Remove Moneris from core

### PR DESCRIPTION
**Pull request by colemanw**
https://issues.civicrm.org/jira/browse/CRM-17004

### CODE REVIEW

File: templates/CRM/Admin/Page/PaymentProcessor.hlp [hunk 1]
```diff
@@ -293,38 +293,6 @@
...
-{/htxt}
-{htxt id='Moneris-test-signature'}
-<p>{ts 1='store3' 2='store1' 3='store2'}All Moneris test transactions are submitted to a shared merchant account. You can use <strong>%1</strong> as your test store ID. '%2' and '%3' also work, but only '%1' accepts recurring payments.{/ts}</p>
-{/htxt}
-{htxt id='Moneris-test-url-site'}
-{ts}The URL for the Moneris TEST Payment server. Use the default value unless otherwise advised by the processor.{/ts}
-{/htxt}
-{htxt id='Moneris-test-url-recur'}
-<p>{ts}The Recurring Payments URL is not currently used for Moneris.{/ts}</p>
-{/htxt}
```
Comment (colemanw) [inline templates/CRM/Admin/Page/PaymentProcessor.hlp]: @adixon this help text doesn't seem to be in your extension yet. Can you copy it there?

**Comment by adixon** [comment]
Yes, please get that code outa core. I don't think I even want much of it in the extension. My goal with the extension is to prop up existing users and those who really want it, not to actually develop it.

**Comment by colemanw** [comment]
jenkins, retest this please

**Comment by colemanw** [comment]
jenkins, retest this please

**Comment by colemanw** [comment]
jenkins, retest this please

**Pull request merged**